src/test/java/no/nav/fo/veilarbregistrering/db/registrering/BrukerRegistreringRepositoryDbIntegrationTest.java

```diff
@@ -221,6 +220,71 @@ public void skal_returnere_empty_naar_ingen_flere_mottatte() {
+                .brukerRegistreringId(lagretRegistrering1.getId())
+                .opprettet(LocalDateTime.now().minusMinutes(5))
+                .status(DOD_UTVANDRET_ELLER_FORSVUNNET)
+                .build();
+        long id1 = brukerRegistreringRepository.lagre(eldsteRegistrering);
+
+        AktiveringTilstand nyesteRegistering = registreringTilstand()
+                .brukerRegistreringId(lagretRegistrering2.getId())
+                .opprettet(LocalDateTime.now().minusMinutes(1))
+                .status(MANGLER_ARBEIDSTILLATELSE)
+                .build();
+        brukerRegistreringRepository.lagre(nyesteRegistering);
+
+        Optional<AktiveringTilstand> nesteTilstand = brukerRegistreringRepository.finnNesteAktiveringTilstandSomHarFeilet();
+
+        assertThat(nesteTilstand.get().getId()).isEqualTo(id1);
+        assertThat(nesteTilstand.get().getBrukerRegistreringId()).isEqualTo(lagretRegistrering1.getId());
+        assertThat(nesteTilstand.get().getStatus()).isEqualTo(DOD_UTVANDRET_ELLER_FORSVUNNET);
+    }
+
+    @Test
+    public void skal_returnere_empty_naar_ingen_flere_som_har_feilet() {
+        OrdinaerBrukerRegistrering registrering1 = gyldigBrukerRegistrering();
+        OrdinaerBrukerRegistrering registrering2 = gyldigBrukerRegistrering();
+        OrdinaerBrukerRegistrering registrering3 = gyldigBrukerRegistrering();
+        OrdinaerBrukerRegistrering lagretRegistrering1 = brukerRegistreringRepository.lagre(registrering1, BRUKER_1);
+        OrdinaerBrukerRegistrering lagretRegistrering2 = brukerRegistreringRepository.lagre(registrering2, BRUKER_1);
+        OrdinaerBrukerRegistrering lagretRegistrering3 = brukerRegistreringRepository.lagre(registrering3, BRUKER_1);
+
+        AktiveringTilstand eldsteRegistrering = registreringTilstand()
+                .brukerRegistreringId(lagretRegistrering1.getId())
+                .opprettet(LocalDateTime.now().minusMinutes(5))
+                .status(ARENA_OK)
+                .build();
+        brukerRegistreringRepository.lagre(eldsteRegistrering);
+
+        AktiveringTilstand nyesteRegistering = registreringTilstand()
+                .brukerRegistreringId(lagretRegistrering2.getId())
+                .opprettet(LocalDateTime.now().minusMinutes(1))
+                .status(UKJENT_BRUKER)
+                .build();
+        brukerRegistreringRepository.lagre(nyesteRegistering);
+
+        AktiveringTilstand midtersteRegistering = registreringTilstand()
+                .brukerRegistreringId(lagretRegistrering3.getId())
+                .opprettet(LocalDateTime.now().minusMinutes(3))
+                .status(UKJENT_TEKNISK_FEIL)
+                .build();
+        brukerRegistreringRepository.lagre(midtersteRegistering);
+
+        Optional<AktiveringTilstand> lagretTilstand = brukerRegistreringRepository.finnNesteAktiveringTilstandSomHarFeilet();
+
+        assertThat(lagretTilstand.isPresent()).isFalse();
```

Comment:
Tror du kan bruke `assertThat(lagretTilstand).hasNoValue()`

Reply:
Hmm, kan ikke finne den

Reply:
OK - er visst i en nyere versjon av biblioteket: `Assertions` i stedet for `Java6Assertions` i samme pakke.

Het heller ikke hasNoValue, men type hasValue, isEmpty osv

Reply:
Får putte det på teknisk gjeld-kontoen. Bør sikkert bytte ut Java6Assertions fortløpende. Det andre gir et mye rikere og mer fluent API.

Reply:
👌